cxx4opencl/address_spaces.txt

```diff
@@ -420,3 +420,27 @@ gengenptr = addrspace_cast<int**>(cnstgenptr); // illegal.
 gengenptr = reinterpret_cast<int**>(locgenptr); // legal.
 gengenptr = reinterpret_cast<int**>(cnstgenptr); // legal.
 ----------
+
+[[remove-addrspace]]
+==== Address space removal
+
+Utility `+__remove address_space+` allows to derive types that have address
+space qualifiers removed. The tool only throws away address space qualifiers,
+therefore, other type qualifiers such as const or volatile remain unchanged.
+
+[source,cpp]
```

Comment:
I feel that the example alone is not enough, we should probably add at least a template class declaration:

`template<class T> struct remove_address_space;`

either following C++ spec style or reference pages:
https://www.cplusplus.com/reference/type_traits/remove_const/

Perhaps we should also point out that it is expected to have similar behavior to `remove_const` and others similar traits from C++17? Then it will be easier to explain and to understand the idea behind this feature.